### DISCUSSION TITLE
Add rel="me" to Mastodon links

### DISCUSSION THREAD

**Discussion by ASTRELION** [JulianPrieber/littlelink-custom#184] [In progress]
It'd be cool if Mastodon links had `rel="me"` in the `a` element for the purpose of [Mastodon's link verification](https://docs.joinmastodon.org/user/profile/#verification).

**Comment by jaschaurbach** [comment]
while we are talking about adding metadata, I would love to add `<link rel="pgpkey" type="application/pgp-keys" href="..." name="My PGP Key" />`. Then LittleLink cold become the hub of all online identities.

**Reply by nobaraos12** [reply]
that would be great!

**Comment by JulianPrieber** [comment]
done
